src/styles/reset.css

```diff
@@ -0,0 +1,4 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
don't do it, it's a bad practice to add `margin: 0;` and `padding: 0;` for tag `*`
Not all tags have default margin and padding, tag `*` works for all tags on the page